modules/islandora_iiif/src/Form/IslandoraIIIFConfigForm.php

```diff
@@ -73,6 +73,14 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#description' => $this->t('Please enter the image server location without trailing slash. e.g. http://www.example.org/iiif/2.'),
       '#default_value' => $config->get('iiif_server'),
     ];
+
+    $form['use_relative_paths'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t("Use relative file paths in manifest."),
+      '#description' => $this->t("Check this if your IIIF Server is configured to access files locally. If unchecked, the absolute URL will be given and the IIIF server will make requests to this site to retrieve images."),
```

Comment:
1 - put in an "advanced" group
2 - mention the need for Cantaloupe to be on the Drupal server.